### PR TITLE
fix: demo seeder config

### PR DESCRIPTION
**Pull request by peterambrus3**

### CODE REVIEW

File: agent/pkg/builder/container/container_builder.go [hunk 1]
```diff
@@ -400,41 +402,54 @@ func prepareImage(dc *DockerContainerBuilder) error {
...
 }
 
-func createNetwork(dc *DockerContainerBuilder) *string {
-	networkName := fmt.Sprintf("%s-network", dc.networkMode)
+func createNetworks(dc *DockerContainerBuilder) []string {
+	var networkIds []string
 
-	filter := filters.NewArgs()
-	filter.Add("name", networkName)
+	for _, network := range dc.networks {
```
Comment (robot9706) [inline agent/pkg/builder/container/container_builder.go:408]: The `network` variable could be called `networkName` so the extra variable could be eliminated.

File: agent/pkg/builder/container/container_builder.go [hunk 2]
```diff
@@ -400,41 +402,54 @@ func prepareImage(dc *DockerContainerBuilder) error {
...
+		}
+
+		networkResult, err := dc.client.NetworkCreate(dc.ctx, networkName, networkOpts)
+		if err != nil {
+			logWrite(dc, fmt.Sprintln("Failed to create network: ", err.Error()))
+			return nil
+		}
+
+		networkID := networkResult.ID
+		networkIds = append(networkIds, networkID)
```
Comment (robot9706) [inline agent/pkg/builder/container/container_builder.go:435]: Could be simplyfied something like `networkIds = append(networkIds, networkResult.ID)`

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .env
 node_modules/
+*.exe
```
Comment (robot9706) [inline .gitignore:3]: If this is used to ignore the build output of the agent then I think instead we could use a .gitingore in the `agent` folder and just ignore the `build/out` path.

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,6 @@
 .env
 node_modules/
+build/out
```
Comment (robot9706) [inline .gitignore:3]: This might need some investigation, since the gitignore in the agent folder already seem to have this.

**Review by robot9706 — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (peterambrus3) [reply .gitignore]: ye, it's not needed, deleted

File: agent/pkg/builder/container/container_builder.go [hunk 5]
```diff
@@ -400,41 +402,51 @@ func prepareImage(dc *DockerContainerBuilder) error {
 	return nil
 }
 
-func createNetwork(dc *DockerContainerBuilder) *string {
-	networkName := fmt.Sprintf("%s-network", dc.networkMode)
+func createNetworks(dc *DockerContainerBuilder) []string {
+	var networkIds []string
```
Comment (m8vago) [inline agent/pkg/builder/container/container_builder.go:406]: networkIDs ?

Comment (peterambrus3) [reply agent/pkg/builder/container/container_builder.go:406]: would you like to rename the variable or ?

**Review by m8vago — APPROVED** [review]

**Pull request merged**